apps/crn-server/src/handlers/teams/algolia-index-team-handler.ts

```diff
@@ -28,7 +28,10 @@ export const indexTeamHandler =
 
       if (team) {
         await algoliaClient.save({
-          data: addTagsFunction(team) as TeamDataObject,
+          data: {
+            ...addTagsFunction(team),
+            members: team.members.length,
+          } as TeamReducedDataObject,
```

Comment:
I've ended up renaming it to `TeamsDataObject` ([rename: TeamReduce -> Teams](https://github.com/yldio/asap-hub/pull/3929/commits/a160c8ed57d2e1a73014ec843f55a7e23cbc102b)) because we already have `ListTeamDataObject` so this

```
export type ListTeamDataObject = ListResponse<TeamListDataObject>;
```

would be confusing